saskatoon/harvest/urls.py

```diff
@@ -9,11 +9,18 @@
 router.register('harvest', api.HarvestViewset, 'harvest')
 router.register('property', api.PropertyViewset, 'property')
 router.register('equipment', api.EquipmentViewset, 'equipment')
-router.register('beneficiary', api.BeneficiaryViewset, 'beneficiary')
+router.register('organization', api.OrganizationViewset, 'organization')
 router.register('community', api.CommunityViewset, 'community')
 router.register('participation', api.RequestForParticipationViewset, 'participation')
 
 urlpatterns = [
+
+    # READ VIEWS
+    path('equipment-points/',
```

Comment:
```suggestion
    path('equipment-point/',
```
use singular, we will most likely add a `equipment-point/create/` and `equipment-point/update` down the road